scripts/ingests/sanghi23/Sanghi23-Ingest.py

```diff
@@ -0,0 +1,118 @@
+#SIMPLE & Astrodb Packages
+# from astrodb_utils.publications import ingest_publication
+from astrodb_utils import load_astrodb, logger
+from astrodb_utils.publications import ingest_publication
+from astrodb_utils.sources import ingest_source, find_source_in_db, ingest_names
+# from simple.utils.spectra import * 
+from simple.schema import *
+from simple.schema import REFERENCE_TABLES
+
+#Additional Packages
+import pandas as pd
+
+# logger.setLevel("DEBUG")
+
+db = load_astrodb("SIMPLE.sqlite", recreatedb=True,  reference_tables=REFERENCE_TABLES)
+
+# Ingest New Sources 
+# ingest_publication(
+#     db,
+#     doi = "10.3847/1538-4357/ad0b12")
+
+newsources = pd.read_csv('missing_sources.csv') # Read in new sources
+for _, source in newsources.iterrows():
+    try :
+        ingest_source(
+            db,
+            source = source['name'],
+            ra = source['ra_j2000_formula'],
+            dec = source['dec_j2000_formula'],
+            reference = source['reference'],
+        )
```

Comment:
You need to pass the column names for ra and dec explicitly because SIMPLE uses the non-default names. (Default is ra_deg and dec_deg).
```suggestion
        ingest_source(
            db,
            source = source['name'],
            ra = source['ra_j2000_formula'],
            dec = source['dec_j2000_formula'],
            reference = source['reference'],
            ra_col_name = 'ra',
            dec_col_name = 'dec'
        )
```